### ISSUE TITLE
Load any image from HiPS database

### ISSUE THREAD

**Issue by vorugantia** [gammapy/gamma-sky#60]
For image selection in the Map View, we should probably find a way to load any HiPS image from the [CDS database](https://github.com/gammapy/gamma-sky/wiki/Datasets#images) without having to define them all in `surveys.ts` like we currently are. We could then implement a `ng2-select` to go through the names of all the HiPS images from this database. This would give the user a much greater selection of maps.

**Comment by cdeil** [comment]
@tboch - Is this something you could implement as a feature in Aladin Lite? Instead of having a pre-defined base map list, have a more fancy select field where there are pre-defined maps, but then one can also search and select from the full list of maps at http://aladin.u-strasbg.fr/hips/list somehow? Or should we try to implement this on our side, for gamma-sky.net?

I think it would be really good to have this, because if someone really wants to see some image from http://aladin.u-strasbg.fr/hips/list and compare it to the gamma-ray sources, there's currently no way and that can be frustrating.

**Comment by tboch** [comment]
yes, having something along these lines is planned, namely being able to :
- choose HiPS to be displayed in the initial select list
- search a HiPS by name/description and add it to the list
